src/models/offer.js

```
@@ -1,47 +1,72 @@
       enum: {
         values: MAIN_ROLE_ENUM,
         message: ENUM_CAN_BE_ONE_OF('author role', MAIN_ROLE_ENUM)
       }
     },
     author: {
       type: Schema.Types.ObjectId,
+      required: [true, FIELD_CANNOT_BE_EMPTY('author')],
       ref: USER
     },
+    category: {
+      type: Schema.Types.ObjectId,
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)